testing/screenshots/index.js

```diff
@@ -46,25 +47,53 @@ function getOptions() {
 }
 
 (async function(){
+    process.argv.forEach((val, index) => {
+        console.log(`${index}: ${val}`);
+      });
+
     const {files, pullRequest} = getOptions();
     console.log("Taking screenshots of", files);
 
     const browser = await puppeteer.launch({ headless: true });
     const screenshots = [];
 
+    var additionalQueryArgs = process.env.ADDITIONAL_QUERY_ARGS
```

Comment:
NITPICK: possibly rename to `GRAFANA_ADDITIONAL_QUERY_ARGS`, as the name doesn't make that clear

Reply:
Good shout.